olcPixelGameEngine.h

```diff
@@ -73,12 +73,35 @@
 	To compile use the command:
 
 	g++ -o YourProgName YourSource.cpp -lX11 -lGL -lpthread -lpng -lstdc++fs -std=c++17
+	
+		Linux Dependencies
+		~~~~~~~~~~~~~~~~~~
+		Besides gcc/g++, you will possibly need a few other libraries installed on the system:
+		
+		- Ubuntu and derivitaves (eg Mint):
+			## GL/gl.H ##
+			sudo apt-get install freeglut3-dev
+			sudo apt-get install binutils-gold mesa-common-dev build-essential libglew1.5-dev libglm-dev
+			## png.H ##
+			sudo apt-get install libxtst-dev libpng++-dev 

```

Comment:
I do not have libxtst-dev on Ubuntu, and the PGE works.
libpng is required, as far as i know.